### ISSUE TITLE
Problem with bootstrap

### ISSUE THREAD

**Issue by sam123sam123** [GrafiteInc/Builder#182]
This probably has something to do with bootstrap v4 and/or laravel 5.5.  I also installed laracogs starter pack.  Pretty sure I followed the instructions correctly.
```
VM1429 app.js:1 Uncaught Error: Module build failed: ModuleBuildError: Module build failed: 
@import "node_modules/bootstrap-sass/assets/stylesheets/bootstrap";
^
      File to import not found or unreadable: node_modules/bootstrap-sass/assets/stylesheets/bootstrap.
Parent style sheet: stdin
      in /var/www/html/laracogs1/resources/assets/sass/app.scss (line 6, column 1)
    at runLoaders (/var/www/html/laracogs1/node_modules/webpack/lib/NormalModule.js:195:19)
    at /var/www/html/laracogs1/node_modules/loader-runner/lib/LoaderRunner.js:364:11
```
That bootstrap-sass directory does not exist.  There does not appear to be a boostrap-sass for v4. 

Running `npm install bootstrap-sass` seems to fix it but that is installing v3 of bootstrap-sass.  It looks like sass is built into v4 so I manually updated the related link in /resources/assets/'sass/app.scss.  That gets rid of the `npm run dev` compile error but it does not seem to be working properly.  A bunch of styling seems to be missing.

**Comment by sam123sam123** [comment]
I fixed it by running npm install bootstrap-sass.  It installs v3 of that package but still seems to work ok with Bootstrap v4.

**Comment by quasiperfect** [comment]
same here

**Comment by mlantz** [comment]
That was a problem with v3 vs v4 for bootstrap, the latest version of Builder now use Bootstrap 4 I only updated that as of 2.3.4, further fixes were implemented in 2.4
